hack/e2e.go

```diff
@@ -18,6 +18,9 @@ limitations under the License.
 package main
 
 import (
+	"github.com/renstrom/dedent"
```

Comment:
There's not a comment up here warning about it, but we have intentionally kept `e2e.go` free of external deps so that it can be `go run` with an airgap.